### DISCUSSION TITLE
Werte von SENEC-Slectrus stimmen nicht mit dem Zählerstand überein

### DISCUSSION THREAD

**Discussion by Opa62** [solectrus/solectrus#3295] [Q&A]
Hallo, ich bin hier zum ersten mal, ich hoffe ihr schreibt mir in Deutsch zurück, den mit Englisch kann ich nix anfangen, bin zu alt da für Danke. 
Die Werte vom Netzbezug und Einspeisung weichen sehr stark vom Stromzähler ab ? Netzbezug war gestern 0,7 kWh, beim Zähler wahren es 2 kWh, bei der Einspeisung wahren es 26,7 kWh, beim Zähler waren es 23 kWh ?? Oder ist das Normal ? 
Die Tageswerte von SENEC.de und Solectrus stimmen bis auf ein Paar Watt überein.
Habt Ihr das selbe Problem ? gibt es hier für eine Lösung?

zu meiner Ausstattung: Besitze den Senec.Home V3 hybrid Duo mit 22 Modulen auf dem Hausdach mit 8,91 kWp und den Speicher mit 
zwei Modulen 2,5 kWh

Vielen Dank

Done

**Comment by ledermann** [comment]
Deutsch ist völlig ok hier :-)

Wenn die Werte von SOLECTRUS mit denen bei mein-senec.de übereinstimmen (das ist auch bei meiner Installation so und war mir immer ein wichtiges Kriterium bei der Entwicklung von SOLECTRUS), ist das Klassenziel erreicht. Der SENEC-Collector fragt die Werte schließlich direkt beim Stromspeicher ab. Dass es überhaupt zu Abweichungen (meist im Promillebereich) kommt, liegt daran, dass die Abfrage nur alle 5 Sekunden erfolgt, der Stromspeicher intern aber genauer messen kann.

Wenn sich bei dir nun eine größere Differenz zum Stromzähler des Hauses ergibt, dann hat das Gründe, die außerhalb des Einflussbereiches von SOLECTRUS liegen. Ärgerlich ist das allemal und sollte so nicht sein. Ich habe das gerade nochmal bei mir geprüft: Die Werte von SENEC/SOLECTRUS und dem Hauszähler liegen äußerst eng beieinander - auch bei großen Zeiträumen. Die Abweichungen sind nicht der Rede wert. Bei dir sind es aber 16%, das ist meiner Ansicht nach viel zu viel.

Ich würde hier zunächst größere Zeiträume vergleichen, also nicht einzelne Tage, sondern einen ganzen Monat oder mehr. Da du als SOLECTRUS-Neuling noch nicht viele Daten haben wirst, solltest du den Vergleich mit den Werten bei mein-senec.de (bzw. der SENEC-App) durchführen.

Wenn die Abweichungen dann immer noch groß sind, wäre das vermutlich eine Sache für den Installateur. Welche Ursachen das haben könnte, kann ich aber nicht abschätzen.

**Comment by norbertle** [comment]
Was da auch in Frage kommt ist ein Ausfall der Internetverbindung.
Dann bekommen das ja Senec und Solectrus nicht mit. Die Anlage produziert aber weiter. Das gibt dann Abweichungen vom Zähler. Weil der erfasst das ja

**Reply by norbertle** [reply]
Update für den Netzbezug könnte das passen aber nicht bei den Einspeisedaten.

**Reply by ledermann** [reply]
Internetausfall halte ich als Ursache für unwahrscheinlich. Soweit ich das beobachtet habe, sammelt der Speicher dann weiterhin alle Messwerte und sendet sie dann später zu SENEC. Auch SOLECTRUS ist es (bei lokaler Installation) egal, ob Internet vorhanden ist oder nicht.

Anders wäre es, wenn der Speicher zeitweise komplett ausgeschaltet wäre. Dann würde der Netzbezug weitergehen und es gäbe eine Abweichung zum Zählerstand von `1.8.0`. Aber die Einspeisung würde dann auch stoppen, eine Abweichung zu `2.8.0` ließe sich damit also nicht erklären.
